### PR TITLE
Refactor legacy

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

File: src/BuildingRegistry.Api.Oslo/Converters/BuildingSorting.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
+namespace BuildingRegistry.Api.Oslo.Converters
```
Comment (ArneD) [inline src/BuildingRegistry.Api.Oslo/Converters/BuildingSorting.cs:1]: Why is this class called BuildingSorting?

File: src/BuildingRegistry.Api.Oslo/Building/Query/BuildingListOsloQuery.cs [hunk 2]
```diff
@@ -1,55 +1,8 @@
 namespace BuildingRegistry.Api.Oslo.Building.Query
```
Comment (ArneD) [inline src/BuildingRegistry.Api.Oslo/Building/Query/BuildingListOsloQuery.cs:1]: Rename class?

**Comment by ArneD** [comment]
Can you check `Api.Legacy` and `Api.Oslo` for the the converters (extension methods) and remove those not used.
For Building and BuildingUnit.

**Review by ArneD — CHANGES_REQUESTED** [review]

File: src/BuildingRegistry.Api.Oslo/Converters/BuildingSorting.cs (hunk 1 above)
Comment (emalfroy) [reply src/BuildingRegistry.Api.Oslo/Converters/BuildingSorting.cs:1]: Wanted to rename another file to building sorting. Guess something went wrong.

**Review by ArneD — APPROVED** [review]

**Pull request merged**